iphone/cli/commands/_build.js

```diff
@@ -6776,9 +6776,27 @@ iOSBuilder.prototype.invokeXcodeBuild = function invokeXcodeBuild(next) {
 	];
 
 	if (this.simHandle) {
+		let dest;
+		const xcodeId = this.xcodeEnv.version + ':' + this.xcodeEnv.build;
+
+		// xcodebuild requires a -destination when building for iOS Simulator and it needs a
+		// simulator that is compatible with the selected Xcode version, so just pick one
+		for (const sim of Object.values(this.iosInfo.simulators.ios)) {
```

Comment:
`this.iosInfo.simulators.ios` is an object with the following structure
```
{
	'13.0': [ // array of sims ],
	'12.4': [ // array of sims ]
}
```

I think this wants to be something like the below
```js
for (const [ runtime, sims ] of Object.entries(this.iosInfo.simulators.ios)) {
	for (const sim of sims) {
		if (sim.supportsXcode[xcodeId]) {
			dest = `platform=iOS Simulator,id=${sim.udid},OS=${appc.version.format(sim.version, 2, 2)}`;
			break;
		}
	}
}
```